### PR TITLE
Added git params to .net assembly (RK-6265)

### PR DESCRIPTION
**Pull request by ElDuderinos**
1. Use https://github.com/MarkPflug/MSBuildGitHash
2. Install it by adding to csproj: 
`<PackageReference Include="MSBuildGitHash" Version="2.0.1" />`
3. Make sure we get all the params:
Add this to get the full commit hash:
`<MSBuildGitHashCommand>git rev-parse HEAD</MSBuildGitHashCommand>`
Add these to get the URL: (this one is hard coded and not retrieved dynamically)
```
    <RepositoryUrl>https://github.com/Rookout/deployment-examples</RepositoryUrl>
    <RepositoryType>git</RepositoryType>
```

See example in program.cs on how to retrieve the data, anyway it should probably not be there as this is just for dev purposes.

### CODE REVIEW

File: dotnet-aspnet/dotnet-core-3/Program.cs [hunk 1]
```diff
@@ -9,13 +9,40 @@
...
+                .Where(c => c.ConstructorArguments.Count >= 2 && c.ConstructorArguments[0].Value as string == "GitRepository")
+                .Select(c => c.ConstructorArguments[1].Value as string)
+                .FirstOrDefault() ?? "??";
+        }
         public static void Main(string[] args)
         {
+            Console.WriteLine(GetBuildHashFromAssembly());
+            Console.WriteLine(GetBuildOriginFromAssembly());
+
 			Rook.RookOptions options = new Rook.RookOptions()
```
Comment (ElDuderinos) [inline dotnet-aspnet/dotnet-core-3/Program.cs:46]: I think we can remove all of this stuff, or at least make it compatible with the format of the data added

Comment (BenTheKlein) [reply dotnet-aspnet/dotnet-core-3/Program.cs:46]: Forgot about this part

**Comment by BenTheKlein** [comment]
@ElDuderinos and i think the docs should link here as well

**Comment by ElDuderinos** [comment]
> @ElDuderinos and i think the docs should link here as well

Sure, add the link to your docs PR then.

**Pull request merged**